backend/src/Entity/ImageEntity.php

```diff
@@ -22,6 +22,9 @@ class ImageEntity
     #[ORM\Column(type: 'string', length: 255)]
     private $imageAim;
```

Comment:
what used for?

Reply:
- change the name to usedAs
- change type to integer.
- we will use numbers as values illustrated in consts class.